api_payments.yaml

```diff
@@ -0,0 +1,286 @@
+swagger: "2.0"
+info:
+  version: 1.0.0
+  title: Proxy PagoPA
+  description: Proxy for PagoPA payment services
```

Comment:
A little more verbose description with the reason for this subsystem (ie. this receives requests from the mobile application backend...)